javascript/javascript-7-oop.md

```diff
@@ -22,9 +22,16 @@
 - [JavaScript 6 - Object Literals Lesson](/javascript/javascript-6-object-literals.md)
 
 ### Motivation
+### Why learn OOP in Javascript?
+- It helps to organize larger projects.
+- It support OOP because it supports inheritance through prototyping as well as properties and methods.
+- Here is the tutorial for [object-oriented Javascript for beginners](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS)
+- Principles of object-oriented programming (aka "OOP") help developers to organize their code. It's really a way of thinking, about how code and data can interact, and how to write code to make it more reusable. It allows developers to organize code to put similar objects under the same category (called "classes"), in order to share code and logic. It also allows other parts of the code to know how to utilize objects, by knowing what's available from that class. 
```

Comment:
Overall this section is good. Please:
1. Move the tutorial line to supplemental materials.  We should not be assigning anything in the Motivation section, just showing examples.
1. Change line 27 to motivate someone who is new to CS... they don't know what inheritance and prototyping are yet. Or just remove it, since line 29 does a pretty good job of that.
1. Be sure to cover these topics: https://github.com/Techtonica/curriculum/issues/1269#issue-580853176